gloom/srv/poetry.lock

```diff
@@ -0,0 +1,38 @@
+[[package]]
+category = "main"
+description = "HTTP REST client, simplified for Python"
+name = "python-http-client"
+optional = false
+python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
+version = "3.3.1"
```

Comment:
We're not using poetry in this project - could you please convert this to pip's normal requirements.txt?